### ISSUE TITLE
[Demande] Fonction signature de post

### ISSUE THREAD

**Issue by FoxMcCloud45** [FranckRJ/RespawnIRC-Android#16]
Bonjour.

Je ne sais pas si cette application est maintenue malgré son retrait du Google Play Store mais je tente quand même.

Y aurait-il moyen d'ajouter un champ de texte permettant :
- soit l'insertion d'une ligne de texte ou d'un paragraphe au choix,
- soit de signer le post avec le pseudonyme en cours ?

Cette demande fait suite à la décision de Webedia d'anonymiser les posts sur Jeuxvideo.com lors d'une citation depuis l'interface Web, ce qui ne permet plus de retracer les réponses à ses posts depuis la barre de recherche.

Merci.

**Comment by FranckRJ** [comment]
Les signatures ça pollue les topics je trouve, j'aime pas vraiment cette solution.

**Comment by FoxMcCloud45** [comment]
Je suis d'accord mais je ne vois guère d'alternative au problème des citations anonymes en conséquence.
